### ISSUE TITLE
[IMPROVE] Forecast Menu - DataFrame Handling When Frequency of Data is not Perfect. 

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#4993]
Currently, the Forecast menu is somewhat limited to the data that can be passed through the models.  It fails when the data is not perfectly even in spacing.  In practice, if the stock does not trade every day because of thin volume, it will fail.   Intraday data also does not work, even on 24/7 time series data like BTC - perfectly spaced at 1 hour intervals.

```
/crypto/load btc -i 60/forecast/rwd btc

Error: Could not observe an inferred frequency. An explicit frequency must be evident over a span of at least 3 consecutive time stamps in the input data. For more information about frequency aliases, read https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases

Error: not enough values to unpack (expected 5, got 0)
```

It might be related to the timestamps being formatted out of the DataFrame:

![Screenshot 2023-05-08 at 10 41 43 AM](https://user-images.githubusercontent.com/85772166/236893222-44f29dc1-a708-4065-b2b6-ccf00ac2d013.png)

An unknown factor would be how timezone localization may impact the dates when a ticker symbol is loaded.

![Screenshot 2023-05-08 at 10 19 39 AM](https://user-images.githubusercontent.com/85772166/236892731-b640a483-7aed-4b6d-aa6b-a103ac9d081b.png)

![Screenshot 2023-05-08 at 10 38 27 AM](https://user-images.githubusercontent.com/85772166/236892738-d5bf159d-0bcb-414d-a383-9f5bec76050a.png)

If I load the Indian stocks at a weekly interval, it works, but not on the daily.

**Comment by deeleeramone** [comment]
Resolved by #5486

**Comment by Stormy333** [comment]
I like this
